app/views/inventories/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<section class="inventory-container">
+  <h1 class="inventory-name">
+    <%= @inventory.name %>
+  </h1>
+  <%= link_to "Add food", new_food_path %>
+  <table class="inventory-table">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Action</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @inventory_foods.each do |food| %>
+        <tr>
+          <td><%= food.name %></td>
+          <td><%= "#{food.quantity} #{food.measurement_unit}" %></td>
+          <td><%= button_to 'Remove', food_path(food), method: :delete %>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</section>
```

Comment:
- [ ] [REQUIRED] Awesome job done team, however, we still need a little tweak to make your design look like the inventory show page provided by the project wireframe.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="559" alt="Screenshot 2023-09-16 at 22 56 27" src="https://github.com/techmoves/recipe-app/assets/58692590/21ae141e-67e8-4f6b-8aa6-753a0c21634b"> | <img width="920" alt="Screenshot 2023-09-16 at 22 56 53" src="https://github.com/techmoves/recipe-app/assets/58692590/c2a07e69-09d0-4d9e-95a9-94f6e2c756b7">